sdk/storage/Azure.Storage.Files.DataLake/src/Models/AccessControlChanges.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// AccessControlChanges contains batch and cumulative counts of operations that change Access Control Lists recursively.
+    /// Additionally it exposes path entries that failed to update while these operations progress.
+    /// </summary>
+    public struct AccessControlChanges
+    {
+        /// <summary>
+        /// An enumerable of path entries that failed to update Access Control List within single batch.
+        /// </summary>
+        public IEnumerable<AccessControlChangeFailure> BatchFailures { get; internal set; }
```

Comment:
@kasobol-msft, @amnguye

Reply:
sounds good.

Reply:
Changed to list.  Do you also want `AccessControlChanges` to be a class instead of a struct?

Reply:
Should we make this an `IList<AccessControlChangeFailure>`, and switch AccessControlChanges.cs to be a class?

Reply:
I think so according to https://www.c-sharpcorner.com/article/what-is-structure-and-when-to-use-in-c-sharp/ ("Use Structure if all member fields are of value type. Use Class if any one member is of reference type.")

Reply:
`IList` is typically done so folks can mutate.  We've been using arrays for output models like this (c.f. `QueueClient. ReceiveAsync`) where we want to make it easy to index but not modify.  We considered `IReadOnlyList` at one point, but it's a bit esoteric so went with array instead.

That's not really good advice for choosing between classes and structs.  Prefer https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct instead, except note that 16 bytes has been changed to 24 bytes in the latest iteration of the design guidelines.  I think it would almost be fine to use a struct here since it's an immutable model type, it's meant to be short lived, and we'll be creating a ton of them...  except it's too large.  Since `AccessControlChangeCounters` are structs, they'll push the size way beyond 24 bytes.

Reply:
Changed BatchFailures to an array.